src/utils/tripHelper.js

```diff
@@ -52,6 +53,29 @@ class tripHelper {
 
     return trip;
   }
+
+  /**
+   *
+   * @param {string} email
+   * @param {string} page
+   * @param {string} limit
+   * @returns {object} allTrips
+   */
+  static async availPendingRequests(email, page, limit) {
+    const pagination = paginate(page, limit);
+    const allTrips = await travelRequests.findAll({
+      where: {
+        status: 'pending', manager: email
+      },
+      attributes: {
+        exclude: ['id', 'userID', 'gender', 'manager', 'createdAt', 'updatedAt']
+      },
+      offset: pagination.offset,
+      limit: pagination.limit,
+    });
+
+    return allTrips;
```

Comment:
@AbdoulNuru just a thought
I think what this will return is the userID and some other raw data we save in the database 
is it possible that you, not just return the requester's id but return his email and name

Reply:
@blaise82 thanks that is a good idea